Content.Server/Interaction/InteractionSystem.cs

```diff
@@ -264,12 +281,29 @@ private bool HandleTryPullObject(ICommonSession? session, EntityCoordinates coor
             return pull.TogglePull(userEntity);
         }
 
-        public async void UserInteraction(IEntity user, EntityCoordinates coordinates, EntityUid clickedUid)
+        /// <summary>
+        ///     Resolves user interactions with objects.
+        /// </summary>
+        /// <remarks>
+        ///     Checks Whether combat mode is enabled and whether the user can actually interact with the given entity. 
+        /// </remarks>
+        /// <param name="altInteract">Whether to use default or alternative interactions (usually as a result of
+        /// alt+clicking). If combat mode is enabled, the alternative action is to perform the default non-combat
```

Comment:
so since combat mode by default does alt interactions on alt+click, does this mean alt+clicking a chemmaster with a beaker inside will open the chemmaster rather than ejecting the beaker (the usual alternative interaction)? if so, this is an okay change i think, in case you really want to use some action but still be in combat mode i guess; but it could be unintuitive

Reply:
might need further discussion in discord about whether it should do normal interactions or the alt interactions while in combat mode

Reply:
i change my mind personally btw, i think it should just do the normal alternative interaction

Reply:
I've upated it to use alt-interact behaviour, regardless of combat mode.